lib/phoenix/view.ex

```diff
@@ -95,54 +95,46 @@ defmodule Phoenix.View do
   can be further configured in the Phoenix application. You can read
   more about format encoders in `Phoenix.Template` documentation.
   """
+  alias Phoenix.{Template}
```

Comment:
I *knew* you would not like this, but I had an overly long line and I had to get rid of it! Don't judge me